sql/moz-fx-data-shared-prod/firefox_desktop_derived/urlbar_events_v1/query.sql

```diff
@@ -1,17 +1,13 @@
-WITH events_unnested AS (
-  SELECT
-    DATE(submission_timestamp) AS submission_date,
-    sample_id,
-    client_info.client_id,
-    timestamp AS event_timestamp,
-    normalized_channel,
-    normalized_country_code,
-    name AS event_name,
-    CASE
+CREATE TEMP FUNCTION enumerated_array(results ARRAY<STRING>) AS (
+  ARRAY(SELECT STRUCT(off AS position, r AS result_type) FROM UNNEST(results) AS r WITH OFFSET off)
```

Comment:
turns out that OFFSET is 0-based. We should use 1-based to align with `selected_position`.

Reply:
thanks for catching that!